src/canister/mod.rs

```diff
@@ -10,10 +10,16 @@ use ic_agent::Agent;
 use serde::Serialize;
 
 use self::utils::get_canisters_list_all;
+#[allow(clippy::all)]
+mod generated;
 
 pub mod reclaim_canisters;
 pub mod snapshot;
 pub mod utils;
+pub use generated::*;
+pub const AGENT_URL: &str = "https://ic0.app";
```

Comment:
is this being used ?

Reply:
nope. 

https://github.com/TwistingTwists/off-chain-agent/blob/fba893df9abf9e46eac73670443dce5141a43061/src/main.rs#L71 
uses the url directly. 

Should we make this configurable via .env file?